### ISSUE TITLE
[Bug] misaligned-continuation-row's false positive after a comment

### ISSUE THREAD

**Issue by MoreFamed** [MarketSquare/robotframework-robocop#818]
### What happened?

![image](https://user-images.githubusercontent.com/82573620/232061389-2d97c7e2-ca07-4c6e-a2be-e3b0fe2df002.png)

### What command/code did you try to run?

robocop, the rule is not configured on my side

### What is the full error message?

Common.resource:90:8 [W] 1015 Each next continuation line should be aligned with the previous one (misaligned-continuation-row)

### What did you expect to happen instead?

I think my code is formatted well, however, on line 90, the warning stated above is displayed. The warning should not be displayed, since the line is aligned with the previous one and with that one before it as well. This uses to happen after comments.

### Operating System

WIndows

### Robocop version

3.0.0

**Comment by mnojek** [comment]
Hey @MoreFamed and thanks for the report!

I took a look at your code and I think you're right that the rule should not be reported.
The reason that it is irrelevantly violated is that the previous line is commented and the indentation is calculated incorrectly.

The fix is on the way and it will be released as Robocop 3.1.0 as soon as possible (probably this week).

**Comment by mnojek** [comment]
FYI The bug is fixed in latest [Robocop 3.1.0 release](https://github.com/MarketSquare/robotframework-robocop/blob/master/docs/releasenotes/3.1.0.rst) :)

You can install the latest available version by:
```
pip install --upgrade robotframework-robocop
```
